[2025-05-25T17:38:44.547+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.create_table scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T17:38:44.570+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.create_table scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T17:38:44.571+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-05-25T17:38:44.591+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): create_table> on 2025-05-01 00:00:00+00:00
[2025-05-25T17:38:44.603+0000] {standard_task_runner.py:57} INFO - Started process 855 to run task
[2025-05-25T17:38:44.611+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'create_table', 'scheduled__2025-05-01T00:00:00+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmp0h3m7534']
[2025-05-25T17:38:44.616+0000] {standard_task_runner.py:85} INFO - Job 20: Subtask create_table
[2025-05-25T17:38:44.723+0000] {task_command.py:410} INFO - Running <TaskInstance: data_pipeline.create_table scheduled__2025-05-01T00:00:00+00:00 [running]> on host bf217478d96c
[2025-05-25T17:38:44.873+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-01T00:00:00+00:00'
[2025-05-25T17:38:44.892+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-05-25T17:38:44.907+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=create_table, execution_date=20250501T000000, start_date=20250525T173844, end_date=20250525T173844
[2025-05-25T17:38:44.941+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-05-25T17:38:44.989+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-25T18:28:40.322+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.create_table scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T18:28:40.339+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.create_table scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T18:28:40.339+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-05-25T18:28:40.360+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): create_table> on 2025-05-01 00:00:00+00:00
[2025-05-25T18:28:40.373+0000] {standard_task_runner.py:57} INFO - Started process 92 to run task
[2025-05-25T18:28:40.384+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'create_table', 'scheduled__2025-05-01T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpv8vtv1dr']
[2025-05-25T18:28:40.390+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask create_table
[2025-05-25T18:28:40.498+0000] {task_command.py:410} INFO - Running <TaskInstance: data_pipeline.create_table scheduled__2025-05-01T00:00:00+00:00 [running]> on host 13c2d9b41bb1
[2025-05-25T18:28:40.636+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-01T00:00:00+00:00'
[2025-05-25T18:28:40.662+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-05-25T18:28:40.686+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=create_table, execution_date=20250501T000000, start_date=20250525T182840, end_date=20250525T182840
[2025-05-25T18:28:40.755+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-05-25T18:28:40.794+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-25T19:22:40.736+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.create_table scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T19:22:40.757+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.create_table scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T19:22:40.758+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-05-25T19:22:40.778+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): create_table> on 2025-05-01 00:00:00+00:00
[2025-05-25T19:22:40.790+0000] {standard_task_runner.py:57} INFO - Started process 78 to run task
[2025-05-25T19:22:40.797+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'create_table', 'scheduled__2025-05-01T00:00:00+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmp0wc5e7m2']
[2025-05-25T19:22:40.803+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask create_table
[2025-05-25T19:22:40.898+0000] {task_command.py:410} INFO - Running <TaskInstance: data_pipeline.create_table scheduled__2025-05-01T00:00:00+00:00 [running]> on host 65cea7b09e55
[2025-05-25T19:22:41.017+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-01T00:00:00+00:00'
[2025-05-25T19:22:41.040+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-05-25T19:22:41.063+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=create_table, execution_date=20250501T000000, start_date=20250525T192240, end_date=20250525T192241
[2025-05-25T19:22:41.130+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-05-25T19:22:41.165+0000] {taskinstance.py:2674} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-05-25T19:41:02.984+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_pipeline.create_table scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T19:41:03.000+0000] {taskinstance.py:1125} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_pipeline.create_table scheduled__2025-05-01T00:00:00+00:00 [queued]>
[2025-05-25T19:41:03.001+0000] {taskinstance.py:1331} INFO - Starting attempt 1 of 2
[2025-05-25T19:41:03.018+0000] {taskinstance.py:1350} INFO - Executing <Task(PythonOperator): create_table> on 2025-05-01 00:00:00+00:00
[2025-05-25T19:41:03.029+0000] {standard_task_runner.py:57} INFO - Started process 318 to run task
[2025-05-25T19:41:03.040+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_pipeline', 'create_table', 'scheduled__2025-05-01T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/data_pipeline.py', '--cfg-path', '/tmp/tmpajd1rftx']
[2025-05-25T19:41:03.045+0000] {standard_task_runner.py:85} INFO - Job 6: Subtask create_table
[2025-05-25T19:41:03.151+0000] {task_command.py:410} INFO - Running <TaskInstance: data_pipeline.create_table scheduled__2025-05-01T00:00:00+00:00 [running]> on host 65cea7b09e55
[2025-05-25T19:41:03.303+0000] {taskinstance.py:1568} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_pipeline' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2025-05-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-05-01T00:00:00+00:00'
[2025-05-25T19:41:03.325+0000] {python.py:183} INFO - Done. Returned value was: None
[2025-05-25T19:41:03.341+0000] {taskinstance.py:1368} INFO - Marking task as SUCCESS. dag_id=data_pipeline, task_id=create_table, execution_date=20250501T000000, start_date=20250525T194102, end_date=20250525T194103
[2025-05-25T19:41:03.411+0000] {local_task_job_runner.py:232} INFO - Task exited with return code 0
[2025-05-25T19:41:03.455+0000] {taskinstance.py:2674} INFO - 1 downstream tasks scheduled from follow-on schedule check
